### PR TITLE
Robot frame work

### PR DESCRIPTION
**Pull request by Jani-siv**
Ready for merge to master. Robot framework template and python unit test template inside robotframework

### CODE REVIEW

**Review by mezi-0 — COMMENTED** [review]
All is done

File: robot/FrameWorkTest.robot [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    ${result} =  Ping Test
+    Should Be True    ${result}
+
+*** Keywords ***
+
+Ping Test
+    [Documentation]    rpiaddress, targetaddress
+    ${result} =      ping local host
+    [Return]    ${result}
+
```
Comment (mezi-0) [inline robot/FrameWorkTest.robot:19]: All is OK

File: robot/Libraries/Framework.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+
+def ping_local_host():
+    retval = os.popen('ping -c 1 127.0.0.1').read()
+    print(retval)
+    for iterator in retval:
+        if iterator.find("0%") < len(iterator):
+            return True
+    return False
+
```
Comment (mezi-0) [inline robot/Libraries/Framework.py:11]: All is OK

File: robot/Libraries/Frameworktest.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+class Frameworktest(unittest.TestCase):
+
+    def test_local_host_ping(self):
+        self.assertTrue(Framework.ping_local_host())
+
+
+if __name__ == '__main__':
+    unittest.main()
+
```
Comment (mezi-0) [inline robot/Libraries/Frameworktest.py:12]: All is OK

File: scripts/mount_image.sh [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+#calculate correct size
+partition1=$(($partition1 * 512))
+sectors1=$(($sectors1 * 512))
+partition2=$(($partition2 * 512))
+
+#mount filesystem
+mount -v -o offset=$partition1,sizelimit=$sectors1 -t vfat $1 /mnt/fat32
+mount -v -o offset=$partition2 -t ext4 $1 /mnt/ext4
+
```
Comment (mezi-0) [inline scripts/mount_image.sh:28]: All is OK

**Review by mezi-0 — CHANGES_REQUESTED** [review]
All is done

File: robot/FrameWorkTest.robot (hunk 1 above)
Comment (Jani-siv) [reply robot/FrameWorkTest.robot:19]: done

**Review by mezi-0 — APPROVED** [review]

**Pull request merged**